play/benchmarker/project/activator-sbt-eclipse-shim.sbt

```diff
@@ -0,0 +1 @@
+addSbtPlugin("com.typesafe.sbteclipse" % "sbteclipse-plugin" % "2.3.0")
```

Comment:
It's to allow eclipse to use the project automatically. This can be removed (it was autogenerated).